src/components/editor/InputLookupQA.jsx

```diff
@@ -84,6 +84,7 @@ class InputLookupQA extends Component {
   render() {
     let authority, subauthority, language
     const lookupConfigs = this.props.lookupConfig
+    const maxRecords = 8
```

Comment:
Can you add this to `src/Config.js` as `maxRecordsForQALookups`?

Reply:
Like this? 
static get maxRecordsForQALookups() {
	  return 8
 }

Reply:
@hudajkhan 	How about:

```javascript
static get maxRecordsForQALookups() {
  return process.env.MAX_RECORDS_FOR_QA_LOOKUPS || 8
}
```

This lets us configure the default via environment variables. Thanks!

Reply:
Added this to Config.js

Reply:
Also resolving because I followed instructions